### ISSUE TITLE
Crash when opening in-game menu (CPY crack)

### ISSUE THREAD

**Issue by tjwoosta** [marcussacana/FAR#10]
07/01/2017 03:18:15.975: -----------------------------------------------------------
07/01/2017 03:18:15.975: [! Except !] 	<< EXCEPTION_ACCESS_VIOLATION >>
07/01/2017 03:18:15.975: -----------------------------------------------------------
07/01/2017 03:18:15.975: [ FaultMod ]  # File.....: 'D:\Games\Installed\Nier Automata\steam_api64.dll'
07/01/2017 03:18:15.975: [ FaultMod ]  * RIP Addr.: steam_api64.dll+000000000000D740h
07/01/2017 03:18:15.975: [StackFrame] <-> Rip=00003b40d740h, Rsp=00000013faa8h, Rbp=00000013fb99h
07/01/2017 03:18:15.975: [StackFrame] >-< Rsi=000000000000h, Rdi=000000000040h
07/01/2017 03:18:15.975: [  GP Reg  ]       rax:     0x00003b40d640
07/01/2017 03:18:15.975: [  GP Reg  ]       rbx:     0x000006b96b20
07/01/2017 03:18:15.975: [  GP Reg  ]       rcx:     0x00003b40d620
07/01/2017 03:18:15.975: [  GP Reg  ]       rdx:     0x0000001d61a0
07/01/2017 03:18:15.975: [  GP Reg  ]        r8:     0x00000013fa38       r9:      0x00000013fb99
07/01/2017 03:18:15.975: [  GP Reg  ]       r10:     0x000000000000      r11:      0x000000000246
07/01/2017 03:18:15.975: [  GP Reg  ]       r12:     0x000000000001      r13:      0x000000000000
07/01/2017 03:18:15.975: [  GP Reg  ]       r14:     0x000000000000      r15:      0x00000013fe40
07/01/2017 03:18:15.975: [ GP Flags ]       EFlags:  0x00010202
07/01/2017 03:18:15.975: -----------------------------------------------------------
07/01/2017 03:18:16.034: [--(Name)--] [!] {         steam_api64.dll}                                         SteamInternal_ContextInit
07/01/2017 03:18:16.080: [-(Source)-] [!] {dxgi.dll} SK_Steam_UserManager::UpdateNumPlayers  <c:\users\andon\source\repos\specialk\src\steam_api.cpp:2330> 
07/01/2017 03:18:16.084: -----------------------------------------------------------

**Comment by marcussacana** [comment]
are you using the stable version?

**Comment by zazomazzo** [comment]
Same Problem here. Tested with stable 0.6.1.2

It starts... I can toggle OSD, but I can't enter In-Game Menu.

**Comment by Gifkun** [comment]
Yep, this happens with the cyp version, it only works if you don't open the menu.

**Comment by Luis-Trindade** [comment]
same here. 
Can anyone copy the far.ini with reshade plugin enabled so I can copy and enable.
Since we can't access the menu there's no way of knowing how to enable/disable certain settings.

**Comment by philocritus** [comment]
same here. 
Can anyone copy the far.ini with reshade plugin enabled so I can copy and enable.
Since we can't access the menu there's no way of knowing how to enable/disable certain settings.

**Comment by tjwoosta** [comment]
The reshade setting is actually in dxgi.ini not far.ini, here are mine though (using the baldman crack with sse)
https://pastebin.com/e0ibSaBd
https://pastebin.com/aA637RTv

**Comment by marcussacana** [comment]
if have problems with the reshade, try read this issue with the google translate
https://github.com/marcussacana/FAR/issues/9

**Comment by philocritus** [comment]
Thanks for your help. I managed to load reshade using the dxgi.ini bit but nier crashes on load.
I also followed marcussacana's link and sugestions but to no avail :(
Has anyone enabled it with cpy crack? or is everybody using baldman's v3?
I fear I'll lose my savegame if I change crack :(

**Comment by zazomazzo** [comment]
Yes, you will lose your savegames. 
Even if you back them up and restore them, they won't be recognized. :(

**Comment by marcussacana** [comment]
the save is easy to convert, with a binary editor find for the cpy long steam id and replace with the baldman long steam id...
Of course, backup it before!

this is the code to convert the long/uint steam id:
```csharp
        private static long ToLongUID(uint UID) => UID + 76561197960265728;
        private static uint ToShortUID(long UID) => (uint)(UID - 76561197960265728);
```

**Comment by marcussacana** [comment]
Srry don´t help guys, i´m with problem in my computer just to update the mod is little hard now...

**Comment by tjwoosta** [comment]
Another thing about getting reshade to work..

If your using sse try disabling the overlay in sse settings, and disable any other overlays you might be using (like nvidia shadowplay, or precisionX overlay, or whatever)

I was getting crash to desktop with the metal gear sound effect when I first tried enabling reshade, it started working properly after I disabled the sse overlay.

**Comment by DStrifer** [comment]
also have problem with CPY. When I try to open FARconfi in game, the game just CTD )

**Comment by DStrifer** [comment]
6.2.11 Same as previous. Crash to desktop when ctrl+shift+backspace. And manual config in FAR,ini not works,

**Comment by marcussacana** [comment]
please test with the baldman version too, backup your save before...

**Comment by DStrifer** [comment]
fuck this crap. yes it works with baldman v3 crack. I did backup but in the same directory as save data just RAR it. Fuck, now the directory is empty........ Now I have a lot of fkn time to wait or start play again but with baldman,,,,,, but i think i`m not, fuck this game until the future official updates. pane in the ass just for 5-10 FPS plus.

**Comment by marcussacana** [comment]
kek, if you give-me your save and the cpy.ini i can try convert the save to the baldman version

**Comment by Forkinator** [comment]
I am getting the same issue, is using the Baldman v3 crack the only way? 

EDIT: Got it to work with Baldman crack, problem is I cannot use my Xbox one controller now... it's not getting any input like it was before when using CPY crack.

**Comment by marcussacana** [comment]
you test use the x360ce? I beat the nier with xinput

**Comment by shinobuowo** [comment]
same problem :(

**Comment by DStrifer** [comment]
When it`s stop crash to DT????  How long this mod will be useless HUD with FPS info and other crap with no impact on game with CPY !?)

**Comment by greedyliz** [comment]
??

**Comment by marcussacana** [comment]
CPY isn't compatible, needs use the baldman version, and is possible convert the savegame, so... isn't needed start from the begin.

**Comment by greedyliz** [comment]
Anyone needing to convert from Baldman to cpy or back can use this. https://github.com/jimmyazrael/NierAutoModSave/raw/master/NierAutoModSave_en_v_0_1.exe Put it in your "Documents\My Games\NieR_Automata\" dir, run it and follow the instructions.

**Comment by greedyliz** [comment]
*baldman

**Comment by Forkinator** [comment]
Thanks greedyliz and Marcus
